src/Mod/Assembly/App/BomObject.cpp

```diff
@@ -252,6 +263,48 @@ void BomObject::addObjectToBom(App::DocumentObject* obj, size_t row, std::string
     }
 }
 
+std::string BomObject::getBomPropertyValue(App::DocumentObject* obj, std::string baseName)
```

Comment:
```suggestion
std::string BomObject::getBomPropertyValue(App::DocumentObject* obj, const std::string& baseName)
```

Do we need to copy `baseName` here?